### PR TITLE
Votes directly to blockchain

### PR DESCRIPTION
**Pull request by Jatewo**
## Description
This pull request includes updates to the `VotingAPI.jsx` and `VerifyKeyPage.jsx` files to integrate a new blockchain API for certain operations. The most important changes include creating a new `blockchainApi` instance and modifying the `vote` and `verifyHash` functions to use this new API.

### Integration of blockchain API:

* [`src/API/VotingAPI.jsx`](diffhunk://#diff-7554713c2cfdd016c5a760e407954119b7b288f99da065952595cd8391fea99eR7-R10): Created a new `blockchainApi` instance using a different base URL (`import.meta.env.VITE_BLOCKCHAIN_API_URL`).
* [`src/API/VotingAPI.jsx`](diffhunk://#diff-7554713c2cfdd016c5a760e407954119b7b288f99da065952595cd8391fea99eL19-R23): Updated the `vote` function to use the `blockchainApi` instance for posting votes instead of the original `api` instance.

### Updates to `VerifyKeyPage`:

* [`src/Screens/VerifyKeyPage.jsx`](diffhunk://#diff-dfbe813082a8d94a8fa5bed054a869ce5f7cde1bc5b8e06a87dcf1a293c53395R11-R14): Created a new `blockchainApi` instance using a different base URL (`import.meta.env.VITE_BLOCKCHAIN_API_URL`).
* [`src/Screens/VerifyKeyPage.jsx`](diffhunk://#diff-dfbe813082a8d94a8fa5bed054a869ce5f7cde1bc5b8e06a87dcf1a293c53395L31-R40): Updated the `verifyHash` function to use the `blockchainApi` instance for verifying keys instead of the original `api` instance.
* [`src/Screens/VerifyKeyPage.jsx`](diffhunk://#diff-dfbe813082a8d94a8fa5bed054a869ce5f7cde1bc5b8e06a87dcf1a293c53395L31-R40): Added optional chaining to handle potential undefined `err.response` in the `verifyHash` function.

### CODE REVIEW

File: src/API/VotingAPI.jsx [hunk 1]
```diff
@@ -4,6 +4,10 @@ const api = axios.create({
     baseURL: import.meta.env.VITE_API_URL,
 });
 
+const blockchainApi = axios.create({
```
Comment (Mafusn) [inline src/API/VotingAPI.jsx:7]: Maybe define this with the other axios api endpoint in the API folder if it still exists

**Review by Mafusn — APPROVED** [review]

File: src/API/VotingAPI.jsx (hunk 1 above)
Comment (Jatewo) [reply src/API/VotingAPI.jsx:7]: No. Does not exist

**Review by MortenMunk — APPROVED** [review]
lgtm

**Review by TheMOHD3886 — APPROVED** [review]
lgtm

**Pull request merged**